src/custom/api/gnosisSafe/index.ts

```diff
@@ -2,15 +2,15 @@ import SafeServiceClient, { SafeInfoResponse, SafeMultisigTransactionResponse }
 import { registerOnWindow } from '@src/custom/utils/misc'
 import { ChainId } from '@uniswap/sdk'
 
-const SAFE_TRANSACTION_SERVICE_URL: Partial<Record<ChainId, string>> = {
```

Comment:
import { SupportedChainId as ChainId } from 'constants/chains' no? or is this because this is safe specific chains?

Reply:
honestly, was cause of 2 things:
- `useActiveWeb3React` return a number as the chainId
- This network thing is now more generic, we accept any number, not just our known chains, if it's unknown it will return `null`